packages/ckeditor5-ui/tests/manual/blocktoolbar/blocktoolbar.html

```diff
@@ -1,45 +1,75 @@
-			<figcaption>Three monks ascending the stairs of an ancient temple.</figcaption>
-		</figure>
-
-		<h3>Appreciation of diversity</h3>
-		<p>
-			Getting used to an entirely different culture can be challenging. While it’s also nice to learn about
-			cultures online or from books, nothing comes close to experiencing cultural diversity in person.
-			You learn to appreciate each and every single one of the differences while you become more culturally fluid.
-		</p>
-
-		<h3>Improvisation</h3>
-		<p>
-			Life doesn't allow us to execute every single plan perfectly. This especially seems to be the case when
-			you travel. You plan it down to every minute with a big checklist. But when it comes to executing it,
-			something always comes up and you’re left with your improvising skills. You learn to adapt as you go.
-			Here’s how my travel checklist looks now:
-		</p>
-
-		<ul>
-			<li>buy the ticket</li>
-			<li>start your adventure</li>
-		</ul>
-
-		<h3>Confidence</h3>
-		<p>
-			Going to a new place can be quite terrifying. While change and uncertainty makes us scared, traveling
-			teaches us how ridiculous it is to be afraid of something before it happens. The moment you face your
-			fear and see there is nothing to be afraid of, is the moment you discover bliss.
-		</p>
+<div class="wrapper-outer">
+				<h3>Appreciation of diversity</h3>
+				<p>
+					Getting used to an entirely different culture can be challenging. While it’s also nice to learn about
+					cultures online or from books, nothing comes close to experiencing cultural diversity in person.
+					You learn to appreciate each and every single one of the differences while you become more culturally fluid.
+				</p>
+
+				<h3>Improvisation</h3>
+				<p>
+					Life doesn't allow us to execute every single plan perfectly. This especially seems to be the case when
+					you travel. You plan it down to every minute with a big checklist. But when it comes to executing it,
+					something always comes up and you’re left with your improvising skills. You learn to adapt as you go.
+					Here’s how my travel checklist looks now:
+				</p>
+	<div class="wrapper">
+					<h3>Appreciation of diversity</h3>
+					<p>
+						Getting used to an entirely different culture can be challenging. While it’s also nice to learn about
+						cultures online or from books, nothing comes close to experiencing cultural diversity in person.
+						You learn to appreciate each and every single one of the differences while you become more culturally fluid.
+					</p>
+
+					<h3>Improvisation</h3>
+					<p>
+						Life doesn't allow us to execute every single plan perfectly. This especially seems to be the case when
+						you travel. You plan it down to every minute with a big checklist. But when it comes to executing it,
+						something always comes up and you’re left with your improvising skills. You learn to adapt as you go.
+						Here’s how my travel checklist looks now:
+					</p>
```

Comment:
Why is it outside any editor?

Reply:
Explained here :point\_right: 35f3bb1#diff-35e1879959ec250ef809949866751a1df0684719286b0b91c9c10b131409eac7